xbmc/guilib/guiinfo/GUIInfoLabels.h

```diff
@@ -154,6 +154,7 @@
 #define NETWORK_DNS2_ADDRESS        197
 #define NETWORK_DHCP_ADDRESS        198
 
+// Kepp musicplayer infolabels that work with offset and poistion togther
```

Comment:
typo: Kepp
typo: poistion
typo: togther

Are you in a hurry? ;-)

Reply:
No, just can't type or read ;)